app/views/find/pages/accessibility.html.erb

```diff
@@ -0,0 +1,39 @@
+<% content_for :page_title, "Accessibility" %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Accessibility statement for Find postgraduate teacher training</h1>
+
+    <p class="govuk-body">This service is run by the Becoming a Teacher team at the Department for Education. We want as many people as possible to be able to use the service. For example, that means users should be able to:</p>
+
+    <ul class="govuk-list govuk-list--bullet">
+      <li>change colours, contrast levels and fonts</li>
+      <li>zoom in up to 300% without the text spilling off the screen</li>
+      <li>navigate most of the service using just a keyboard</li>
+      <li>navigate most of the service using speech recognition software</li>
+      <li>listen to most of the content on the service using a screen reader (including the most recent versions of JAWS, NVDA and VoiceOver)</li>
+    </ul>
+
+    <p class="govuk-body">We’ve also made the text on the service as simple as possible to understand.</p>
+
+    <p class="govuk-body"><%= govuk_link_to "AbilityNet", "https://mcmw.abilitynet.org.uk/" %> has advice on making your device easier to use if you have a disability.</p>
```

Comment:
Hmm, it might be a good idea to put external links in the translations and prefix them with url, so they are easy to audit in the future.

Reply:
Something like this 
<img width="489" alt="image" src="https://user-images.githubusercontent.com/50492247/184600143-8655b5c7-9fbc-4521-9c25-e9186d9255b9.png">

Reply:
WDYT?